src/TodoAddForm.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export default function TodoAddForm({ newTodo, onSubmit, onChange }) {
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    onSubmit();
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input
+        type="text"
+        placeholder="할 일을 입력해 주세요"
+        value={newTodo}
+        onChange={(e) => onChange(e.target.value)}

```

Comment:
```suggestion
        onChange={onChange}
```
라고만 처리하고 App의 핸들러에서 e를 매개변수로 받아와 처리하는건 어떨까요?

Reply:
이 부분은 조금 더 생각해보겠습니다. 의도는 확실히 파악된 것 같습니다.

저는 '상태 업데이트' 와 '이벤트 핸들링' 을 분리 시켜서 상태업데이트는 App 에서 수행하고 
하위 컴포넌트에서 이벤트 핸들링을 할때, 상태업데이트를 의존성을 불러와서 처리하는 것을 관심사의 분리로 생각했고,

트레이너님은 둘을 분리시키지 않고 '이벤트 핸들링' 함수 로직을 모두 App에 모으고,
하위 컴포넌트에서, 이벤트 핸들러를 의존성으로 처리하는것을 관심사의 분리로 생각하신것 같습니다.

확실히, 이벤트 핸들러 로직이 한군데 모여 있는편이 더 심플할 것 같다는 생각이 듭니다.
굳이 상태 업데이트와 이벤트 핸들링을 분리해서 생각할 이유가 있나 생각도 들구요.
조금더 생각해 보겠습니다. ㅎ
감사합니다!